wpiunits/src/test/java/edu/wpi/first/units/MeasureTest.java

```diff
@@ -194,6 +202,30 @@ void testDivideMeasure() {
     result = m9.divide(m10);
     assertEquals(result.magnitude(), 2);
     assertEquals(result.unit(), Units.Value);
+    // Dimensionless divided by Velocity
+    var m11 = Units.Value.of(8);
+    var m12 = Units.Meters.per(Units.Second).of(4);
+    result = m11.divide(m12);
+    assertEquals(result.magnitude(), 2);
+    assertEquals(result.unit(), Units.Seconds.per(Units.Meter));
+    // Dimensionless divided by Per
+    var m13 = Units.Value.of(6);
+    var m14 = Units.Volts.per(Units.Meter).of(2);
+    result = m13.divide(m14);
+    assertEquals(result.magnitude(), 3);
+    assertEquals(result.unit(), Units.Meters.per(Units.Volt));
+    // Dimensionless divided by Per<Time, U>
+    var m15 = Units.Value.of(10);
+    var m16 = Units.Milliseconds.per(Units.Meter).of(2);
+    result = m15.divide(m16);
+    assertEquals(result.magnitude(), 5);
+    assertEquals(result.unit(), Units.Meters.per(Units.Millisecond));
+    // Time divide
+    var m17 = Units.Meters.of(8);
+    var m18 = Units.Seconds.of(2);
+    result = m17.divide(m18);
+    assertEquals(result.magnitude(), 4);
+    assertEquals(result.unit(), Units.Meters.per(Units.Second));
```

Comment:
I suggest splitting the individual scenarios into separate tests, this one test method is getting too large.